test/Chainweb/Test/Pact/PactSingleChainTest.hs

```diff
@@ -451,6 +457,72 @@ compactionIsIdempotent rdb =
                   putStrLn ""
       assertFailure "pact state check failed"
 
+compactionTransactionIndex :: ()
+  => RocksDb
+  -> TestTree
+compactionTransactionIndex rdb =
```

Comment:
TODO: add another test to attempt to submit a duplicate tx after compaction and see if it is rejected.